man/systemd.nspawn.xml

```diff
@@ -224,6 +224,19 @@
       </varlistentry>
 
       <varlistentry>
+        <term><option>KillSignal=</option></term>
+
+        <listitem><para>Specify the process signal to send to the
+        container's PID 1 when nspawn itself receives SIGTERM, in
+        order to trigger an orderly shutdown of the
+        container. Defaults to SIGRTMIN+3 if <option>Boot=</option>
+        is used (on systemd-compatible init systems SIGRTMIN+3
+        triggers an orderly shutdown). Takes a signal name like
+        <literal>SIGHUP</literal>, <literal>SIGTERM</literal> or
+        similar as argument.</para></listitem>
+      </varlistentry>
+
+      <varlistentry>
```

Comment:
Please replace that last sentence with `For a list of valid signals, see signal(7).`